internal/jennies/java/builders.go

```diff
@@ -24,7 +24,9 @@ func parseBuilders(config Config, context languages.Context, formatter *typeForm
 		if _, ok := b[builder.Package]; !ok {
 			b[builder.Package] = map[string]ast.Builder{}
 		}
-		b[builder.Package][builder.Name] = builder
+
+		obj, _ := context.LocateObject(builder.For.SelfRef.ReferredPkg, builder.For.SelfRef.ReferredType)
+		b[builder.Package][obj.Name] = builder
```

Comment:
If you only need to access the object's name, you don't need to locate it :)

This should be equivalent:
```suggestion
		b[builder.Package][builder.For.SelfRef.ReferredType] = builder
```

Reply:
Right!